app/models/ability.rb

```diff
@@ -4,6 +4,7 @@ class Ability
   def initialize(user)
     if user
       can %i(create read destroy update), Wishlist, user_id: user.id
+      can :manage, Item, :wishlist => { :user_id => user.id }
```

Comment:
Guys, it has label in progress beacuse I'm still in process of refactoring :) rubocop points out array syntax and all that jazz, I'll correct everything before changing label to ready to check. Now I'm just trying to make tests to pass